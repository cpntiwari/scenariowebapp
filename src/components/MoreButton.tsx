import React from "react";
import { createStyles, makeStyles, Theme } from "@material-ui/core/styles";
import { Button } from "@material-ui/core";
import { Palette } from "@jda/lui-common-component-library";
const useStyles = makeStyles((theme: Theme) =>
  createStyles({
    root: {
      "& > *": {
        margin: theme.spacing(0),
        fontSize: theme.typography.pxToRem(12),
        fontWeight: theme.typography.fontWeightBold,
        color: Palette.colors.green["700"],
        textDecoration: "none",
        cursor: "pointer"
      },
    },
  })
);

export default function OutlinedButtons() {
  const classes = useStyles();
  return (
    <div className={classes.root}>
      <Button variant="outlined" color="primary" >More Details</Button>
    </div>
  );
}
