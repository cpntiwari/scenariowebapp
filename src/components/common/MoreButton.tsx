import React from "react";
import { createStyles, makeStyles, Theme } from "@material-ui/core/styles";
import { Button } from "@material-ui/core";
import { Palette } from "../../themes/Palette";
const useStyles = makeStyles((theme: Theme) =>
  createStyles({
    root: {
      "& > *": {
        margin: theme.spacing(0),
        fontSize: theme.typography.pxToRem(12),
        fontWeight: theme.typography.fontWeightBold,
        color: Palette.colors.blue["700"],
        textDecoration: "none",
        cursor: "pointer"
      },
    },
  })
);

export default function OutlinedButtons() {
  const classes = useStyles();
  return (
    <div className={classes.root}>
      <Button variant="outlined" color="primary" >More Details</Button>
    </div>
  );
}
