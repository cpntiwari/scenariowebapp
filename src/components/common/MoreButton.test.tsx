import React from "react";
import MoreButton from "./MoreButton";
import { shallow } from "enzyme";
let wrapper: any;

beforeEach(() => {
  wrapper = shallow(<MoreButton />);
});

describe("More Button Component Rendering", () => {
  test("should test More Button component", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
