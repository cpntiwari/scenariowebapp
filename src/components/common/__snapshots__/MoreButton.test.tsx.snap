// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`More Button Component Rendering should test More Button component 1`] = `
<div
  className="makeStyles-root-1"
>
  <WithStyles(ForwardRef(Button))
    color="primary"
    variant="outlined"
  >
    More Details
  </WithStyles(ForwardRef(Button))>
</div>
`;
